server-base/src/main/java/org/apache/kylin/rest/job/KylinHealthCheckJob.java

```diff
@@ -0,0 +1,298 @@
+/*
```

Comment:
Hi, @lshmouse , I found this java file is not formatted as expected. It looks like it is indented with two blank and it is different with other Java file, maybe you should check **Setup IDE code formatter** in http://kylin.apache.org/development/dev_env.html.